doc/3/controllers/document/update/snippets/update.test.yml

```diff
@@ -0,0 +1,10 @@
+name: document#update
+description: Update a document
+hooks:
+  before: |
+    curl -XDELETE kuzzle:7512/nyc-open-data
+    curl -XPOST kuzzle:7512/nyc-open-data/_create
+    curl -XPUT kuzzle:7512/nyc-open-data/yellow-taxi
+  after:
+template: default
+expected: "id: 'some-id'" 
```

Comment:
I'm not sure that this test can pass. Can you add documentation snippet tests to travis to make sure of that?